database/models.go

```diff
@@ -0,0 +1,44 @@
+package database
+
+import (
+	"database/sql"
+	"time"
+
+	"github.com/google/uuid"
+)
+
+type Election struct {
+	ID         uuid.UUID    `gorm:"primaryKey"`
```

Comment:
Yes, it was a fairly deliberate choice. Both of the points you made are correct, and I also want to minimise the amount of information that an admin could use for trying to make connections to what specific people have voted for. 

Still, that reasoning only really applies to `Vote`, `Ranking` and `CastedVote`, so maybe it could be added for the rest. And the argument can be made that `CreatedAt`, `UpdatedAt` and `DeletedAt` could still be useful (or maybe even essential?) to be able to properly verify the election. I do remember having a discussion with Gabbe regarding this specific point.

Reply:
Makes sense, I was kind of thinking about data revisorer might need to see.